### PR TITLE
Seq Diagram Fixes

### PR DESCRIPTION
**Pull request by dhruvigoyal25**
This PR contains the following fixes:
1. State change for a pause of sequence logs
2. Export takes a lot of time.

### CODE REVIEW

**Review by dilallkx — CHANGES_REQUESTED** [review]
Please address the provided comments.
Thanks!

File: go-packages/meep-metrics/metric-store.go [hunk 1]
```diff
@@ -309,44 +311,110 @@ func (ms *MetricStore) GetInfluxMetric(metric string, tags map[string]string, fi
...
-	if len(response.Results) > 0 && len(response.Results[0].Series) > 0 {
-		row := response.Results[0].Series[0]
-		for _, qValues := range row.Values {
-			rValues := make(map[string]interface{})
-			for index, qVal := range qValues {
-				rValues[row.Columns[index]] = qVal
+	return valuesArray, nil
+}
+
+func (ms *MetricStore) GetTagStr(inputTagStr string, timeStart string, timeStop string, duration string) string {
```
[the remark this answers is not in the document]
Comment (dhruvigoyal25) [reply go-packages/meep-metrics/metric-store.go]: Addressed.

File: go-packages/meep-metrics/metric-store.go [hunk 2]
```diff
@@ -309,44 +311,110 @@ func (ms *MetricStore) GetInfluxMetric(metric string, tags map[string]string, fi
 		}
 		tagStr += ")"
 	}
+
+	valuesArray := []map[string]interface{}{}
```
[the remark this answers is not in the document]
Comment (dhruvigoyal25) [reply go-packages/meep-metrics/metric-store.go]: Addressed.

File: go-packages/meep-metrics/metric-store.go [hunk 3]
```diff
@@ -309,44 +311,110 @@ func (ms *MetricStore) GetInfluxMetric(metric string, tags map[string]string, fi
...
+		response, err := (*ms.influxClient).Query(q)
+		if err != nil {
+			log.Error("Query failed with error: ", err.Error())
+			return values, err
+		}
 
-	query := "SELECT " + fieldStr + " FROM " + metric + " " + tagStr + " ORDER BY desc" + countStr
-	log.Debug("QUERY: ", query)
+		// Process response
+		if len(response.Results) > 0 && len(response.Results[0].Series) > 0 {
```
[the remark this answers is not in the document]
Comment (dhruvigoyal25) [reply go-packages/meep-metrics/metric-store.go:358]: Addressed.

**Review by dilallkx — APPROVED** [review]

**Pull request merged**